R/parameters_table.R

```diff
@@ -224,10 +228,11 @@ parameters_table <- function(x, pretty_names = TRUE, stars = FALSE, digits = 2,
   }
 
   x[std_cols] <- insight::format_value(x[std_cols], digits = digits)
-  names(x)[names(x) == std_cols] <- "Beta (std.)"
+  names(x)[names(x) == std_cols] <- .replace_words(std_cols, "Std_", "Std. ")
 
   if (!is.null(std_cis) && length(std_cis)) {
-    std_cis_replacement <- gsub("^Std_Coefficient(.*)", "Beta\\1", std_cis)
```

Comment:
I think the coefficient name for standardized is quite long, so I thought using beta here.

Reply:
I agree it's long, but then it should be consistent. Another option is to use abreviations (Coef.) or something. But it's not that big of a deal if it's long imo (but it's definitely a nogo to swap the name, especially for some random unrational, implicit, illogical and inconsistent convention - I'll die on that hill if I need to 😁)

Reply:
Than coef.? If we compare multiple models in one table, column names need to be short.

Reply:
Then, `Coef.` is a variable alternative